File: Ohtap E Commerce Domain Models/Development/PHP/Objects/ProductLabels/IProductLabelData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Promotions;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IProductLabelData extends IProductLabelLabel,IProductLabelValue,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}

File: Ohtap E Commerce Domain Models/Development/NetCore/Models Objects/Models Objects/ProductLabels/IProductLabelData.cs
using Ohtap.ECommerce.Domain.Models.Objects.Datasets;
using Ohtap.ECommerce.Domain.Models.Objects.Labels;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;


namespace Ohtap.ECommerce.Domain.Models.Objects.ProductLabels
{
    public interface IProductLabelData : CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IProductLabelValue
    {
        ILabelData Label { get; set; }
        IDataItemList<IDataItemData> Dataset { get; set; }
    }
}
